.github/workflows/omni.yml

```diff
@@ -59,29 +59,26 @@ jobs:
         const DEFAULT_TEST_GROUP = process.env.MATRIX_DEFAULT_TEST_GROUP || 'fast';
 
         const JDK_OPTIONS = [
-            { group: 'Zulu', version: '8', lts: true, distribution: 'zulu' },
-            { group: 'Zulu', version: '11', lts: true, distribution: 'zulu' },
-            { group: 'Zulu', version: '15', lts: false, distribution: 'zulu' },
+            { group: 'Zulu', version: '17', lts: true, distribution: 'zulu' },
+            { group: 'Zulu', version: '21', lts: false, distribution: 'zulu' },
```

Comment:
This should be a LTS version, shouldn't it?

Reply:
aye, yes it should, thanks for the review